src/computedFn.ts

```diff
@@ -67,7 +67,10 @@ export function computedFn<T extends (...args: any[]) => any>(
         if (entry.exists()) return entry.get().get()
         // if function is invoked, and its a cache miss without reactive, there is no point in caching...
         if (!opts.keepAlive && !_isComputingDerivation()) {
-            if (!memoWarned && _getGlobalState().computedRequiresReaction) {
+            if (
+                !memoWarned &&
+                (opts.requiresReaction ?? _getGlobalState().computedRequiresReaction)
```

Comment:
Can we use `??`? 
Otherwise I think this should be `typeof opts.requiresReaction === "boolean" ? opts.requiresReaction : _getGlobalState().computedRequiresReaction`